test/controllers/actions/FeatureFlagActionSpec.scala

```diff
@@ -17,14 +17,16 @@
 package controllers.actions
 
 import base.SpecBaseWithApplication
+import com.google.inject.Inject
 import controllers.actions.FeatureFlag.{TopupJourneyFlag, VariableJourneyFlag}
 import controllers.routes
+import handlers.ErrorHandler
 import play.api.mvc.Results
 import play.api.test.Helpers._
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class FeatureFlagActionSpec extends SpecBaseWithApplication {
+class FeatureFlagActionSpec @Inject()(eh: ErrorHandler) extends SpecBaseWithApplication {
```

Comment:
If these works is fine, but I think we can wire this with `app.injector.instanceOk[ErrorHandler]` directly inside the body of the test.